2_observations.yml

```diff
@@ -0,0 +1,76 @@
+target_default: 2_observations
+
+packages:
+  - sf
+  - tidyverse
+  - lwgeom
+  - smoothr
+  - igraph
+  - purrr
+
+sources:
+  - 2_observations/src/crosswalk_functions.R
+  - 2_observations/src/data_munge_functions.R
+
+targets:
+  2_observations:
+    depends:
+     - 2_observations/out/basin_temp_data.rds.ind
+     
+  # temperature sites
+  2_observations/in/all_sites.rds:
+    command: gd_get('2_observations/in/all_sites.rds.ind')
+    
+  # munged temperature data
+  2_observations/in/daily_temperatures.rds:
+    command: gd_get('2_observations/in/daily_temperatures.rds.ind')
+     
+  2_observations/out/crosswalk_site_reach.rds.ind:
```

Comment:
I got this far without having to rebuild anything, but scipiper wanted to rebuild this target.
```r
> which_dirty('2_observations/out/crosswalk_site_reach.rds.ind')
[1] "2_observations/out/crosswalk_site_reach.rds.ind"
```

I see the .ind file in my directory and confirmed that its build/status file already exists by running `scipiper:::locate_build_status_yml('2_observations/out/crosswalk_site_reach.rds.ind')`. So that's not it.

Instead, the issue appears to be with `subset_closest.R` changing, which I can see by running `why_dirty`:
```r
> why_dirty('2_observations/out/crosswalk_site_reach.rds.ind')
Since the last build of the target '2_observations/out/crosswalk_site_reach.rds.ind':
  * the dependency '2_observations/src/subset_closest.R' has changed
# A tibble: 7 x 8
  type    name                 hash_old         hash_new         hash_mismatch dirty dirty_by_descent current
  <chr>   <chr>                <chr>            <chr>            <lgl>         <lgl> <lgl>            <lgl>  
1 target  2_observations/out/… 6b09d1b0076da8d… 6b09d1b0076da8d… FALSE         TRUE  FALSE            FALSE  
2 depends 2_observations/in/a… 460e7b4b92fb885… 460e7b4b92fb885… FALSE         FALSE FALSE            TRUE   
3 depends 1_network/out/bound… bfeabb362af0ed4… bfeabb362af0ed4… FALSE         FALSE FALSE            TRUE   
4 depends 1_network/out/netwo… a2cb3b67100dcba… a2cb3b67100dcba… FALSE         FALSE FALSE            TRUE   
5 depends 2_observations/src/… 8180c97825dfb0a… 7750c466becd936… TRUE          FALSE FALSE            TRUE   
6 fixed   NA                   8c7c06704515b69… 8c7c06704515b69… FALSE         FALSE FALSE            TRUE   
7 functi… crosswalk_sites_to_… a73fd312dfd6035… a73fd312dfd6035… FALSE         FALSE FALSE            TRUE 
```

Two possibilities are that (1) you modified subset_closest.R after building this target, or (2) you're on a windows machine and I'm on a mac, so our line endings are different, and because this code file is linked through `depends` and `source` rather than `sources`, remake is unwilling to overlook that difference. I think 2 is more likely, though you can rule out 1 if you like by attempting to build this target locally (if 1 is not true, the target will be current already).